File: logic build problem/easy problems/Typescript/reverseDigit.ts
function reverseDigit(num:number):number{
    let rev:number = 0;
    let lastDigit:number = 0;
    while (num!=0){
        lastDigit = num%10
        rev = (rev*10)+lastDigit
        num = Math.floor(num/10)
    }
    return rev
}

console.log(reverseDigit(126))

File: logic build problem/easy problems/python/reverseDigit.py
def reverseDigit(num:int)->int:
    reverseDigit:int=0
    while num!=0:
        lastDigit = num % 10
        reverseDigit = (reverseDigit * 10) + lastDigit
        num = num // 10
    return reverseDigit
    
print(reverseDigit(386))
